src/main/scala/Matrix.scala

```diff
@@ -0,0 +1,22 @@
+class
+Matrix {
+
+  val rand = scala.util.Random
+
+  def printRandoms = {
+    while(true){
+      println()
+      for (i <- 0 to rand.nextInt(500))
+        print(rand.nextPrintableChar())
+    }
+  }
+
+  def printRandomsRec (counter: Int = 1) : Unit = {
```

Comment:
could rework this into one without having a for loop, if you had two counters I think